integration/spark/integrations/container/pysparkCTASWithColumnLineageEnd.json

```diff
@@ -0,0 +1,70 @@
+{
+  "eventType" : "COMPLETE",
+  "job" : {
+    "namespace" : "testCreateAsSelectAndLoad",
+    "name" : "open_lineage_integration_cta_s_load.execute_create_hive_table_as_select_command"
+  },
+  "inputs" : [ {
+    "namespace" : "file",
+    "name" : "/tmp/ctas_load/temp",
+    "facets" : {
+      "dataSource" : {
+        "name" : "file",
+        "uri" : "file"
+      },
+      "schema" : {
+        "fields" : [ {
+          "name" : "a",
+          "type" : "long"
+        }, {
+          "name" : "b",
+          "type" : "long"
+        } ]
+      }
+    }
+  } ],
+  "outputs" : [ {
+    "namespace" : "file",
+    "name" : "/tmp/ctas_load/tbl1",
+    "facets" : {
+      "dataSource" : {
+        "name" : "file",
+        "uri" : "file"
+      },
+      "schema" : {
+        "fields" : [ {
+          "name" : "a",
+          "type" : "long"
+        }, {
+          "name" : "b",
+          "type" : "long"
+        } ]
+      },
+      "lifecycleStateChange" : {
+        "lifecycleStateChange" : "CREATE"
+      },
+      "columnLineage": {
+        "fields": {
+          "a": {
+            "inputFields": [
+              {
+                "namespace": "file",
+                "name": "/tmp/ctas_load/temp",
```

Comment:
Both `namespace` and `name` describe input datasets. 
Yes, you're right: `inputNamespace` and `inputName` would be more explicit.
I wouldn't like to add this to this PR due to its size. 
However, I can prepare another one just after this to rename it. 
What do you think?

Reply:
I think combination of `namespace` and `name` are describing `dataset` everywhere else in the spec. 

Renaming `name` to `dataset` would be confusing, since `name` does not specify dataset by itself - only in combination with `namespace`. 

I think adding nesting might do what you want to do, but don't think it would add clarity.

```json
"inputFields": [{
    "dataset": {
        "namespace": "file",
        "name": "/tmp/ctas_load/temp",    
    },
    "field": "a"
}]
```